### ISSUE TITLE
Finally profitable digit differ

### ISSUE THREAD

**Issue by delano24** [Fruitfella/ROBOT-binary#425]
![Screenshot_20190621_225234](https://user-images.githubusercontent.com/37808777/59931573-d8278b00-9477-11e9-8ec0-777966fadb17.jpg)

**Comment by delano24** [comment]
![Screenshot_20190621_233203](https://user-images.githubusercontent.com/37808777/59934031-32771a80-947d-11e9-8673-fb2311f5e44d.jpg)

**Comment by delano24** [comment]
![Screenshot_20190621_233909](https://user-images.githubusercontent.com/37808777/59934295-cea12180-947d-11e9-8147-db0e23c70a82.jpg)

**Comment by delano24** [comment]
Just comment below..

**Comment by delano24** [comment]
![Screenshot_20190621_234207](https://user-images.githubusercontent.com/37808777/59934481-38b9c680-947e-11e9-8432-80726f4dcd8a.jpg)
Its freeeee... Gratissss........ 
Just please help me to  make the mm .. i have the nice mm
Cuma bantu saya membuat mmnya. Mmnya mantap...

**Comment by delano24** [comment]
Desc bot standard no MM... 
Its safe no mm......

**Comment by pieter1987** [comment]
You can contact me on pdmliebenberg7@gmail.com

**Comment by nobleb2007** [comment]
Nice results, how can I benefit from it oo.

**Comment by rjaliya** [comment]
Send me a copy for testing rjaliya791@gmail.com

**Comment by nobleb2007** [comment]
Kindly send me the bot. Please nobleb2007@gmail.com

**Comment by delano24** [comment]
@nobleb2007@gmail.com can help for bulit my MM?
@pieter1987 can help for bulit my MM?
@rjaliya i will sent to.. please help to mkae my MM. I will give to you free

**Comment by pieter1987** [comment]
Yes i can help you with MM please send it along to my email pdmliebenberg24@gmail.com then i can have a look at it and help you with MM

**Comment by delano24** [comment]
![Screenshot_20190623_182740](https://user-images.githubusercontent.com/37808777/59974945-c7485800-95e4-11e9-8b4f-801bf3a754b3.jpg)
NO SCAMMMMM....

**Comment by delano24** [comment]
@pieter1987 ok ill sent the rule for my mm..

**Comment by lasaidhari** [comment]
@delano24 send saidforex26@gmail.com

**Comment by delano24** [comment]
lasaidhari ill sent but u must help me to built bot with my mm

**Comment by lasaidhari** [comment]
yes I will help your mm

**Comment by delano24** [comment]
![Screenshot_20190624_094458](https://user-images.githubusercontent.com/37808777/59985533-d0790980-9664-11e9-99ce-c6f3174d31a6.jpg)
Result morning

**Comment by lasaidhari** [comment]
send

**Comment by lasaidhari** [comment]
krim

**Comment by delano24** [comment]
Indo????

**Comment by lasaidhari** [comment]
ya indonesia

**Comment by tryx68** [comment]
Plse sent to tryx1969@gmail.com  , thks

**Comment by rjaliya** [comment]
> @[nobleb2007@gmail.com](mailto:nobleb2007@gmail.com) can help for bulit my MM?
> @pieter1987 can help for bulit my MM?
> @rjaliya i will sent to.. please help to mkae my MM. I will give to you free

What is MM stands for?

**Comment by ceakuk** [comment]
you can never predict random numbers. you might see profit  for  but in long run, it will be wipped off by the martingale  in a digitdiff, a win rate  should never be more than 9 out of 10. its pure mathematics

**Comment by rjaliya** [comment]
Is there any profitable bot exists in this world?

**Comment by ceakuk** [comment]
@delano24 
i wanted to test your claims, so i emailed you but the email does not go through. are you legit?

**Comment by choirul17** [comment]
Gan Boleh coba bot nya Send crl.amin@gmail.com please gan

**Comment by delano24** [comment]
@ceakuk until now. Iddnt found 3 loss row. . Dont use marti. yesss im legit.. just sent me to my another gmail bernandodelano000@gmail.com

@choirul iya saya mau kirim tpi bisa dibantu untuk membuat mm yang saya minta

@rjaliya i didnt recommend use for martingale. I recommend use my MM for best recover

**Comment by delano24** [comment]
@lasaidhari iya bro mohon dibntu buat mm. Saya minta no wa aja deh

**Comment by delano24** [comment]
@tryx68 ill sent it. Its free.
Just help me for built the mm...

**Comment by delano24** [comment]
![15615747043341931990612061054790](https://user-images.githubusercontent.com/37808777/60206221-37710b00-987d-11e9-9241-2a16577c1f03.jpg)
So far until now ididnt found 2 times a row loss.. but idont recoment with mm martix10-11.. it will make you bankrupt

**Comment by lasaidhari** [comment]
@delano24 ni no wa saya 082290076815

**Comment by rakage** [comment]
Halo bisa tolong kirim ke rakageeee@gmail.com
Terima kasih:)

**Comment by choirul17** [comment]
@delano24 mm yang seperti bagaimana bro ??
Sini bro saya edit

**Comment by choirul17** [comment]
@delano24 jelasin mm yang anda inginkan bro

**Comment by upin1506** [comment]
boleh nyoba botnya om? bisa kirim ke syaiupin40@gmail.com thank's

**Comment by lasaidhari** [comment]
![Mts avg rsi sudut v1 1](https://user-images.githubusercontent.com/34808023/60640476-33b22b00-9e1f-11e9-8daa-d88fea1a31a9.gif)

Pada Bot ini mengunakan indikator RSI 14 dan sudut kemiringan RSI. bot juga telah disertakan dengan pengaturan martinggale MTS dengan pengaturan yang dapat digunakan . 
1. total saldo / stop loss 
2. target profit 
3. kapan mulai martinggale 
4. martingale 1-100/refres/level/back/reset 
5. maksimal martinggale loss setiap level 
6. maksimal martinggale level loss 
7. kapan mulai martingale pengembalian saldo yang hilang 
8. martinggale pengembalian saldo yang hilang 
9. refresh stake setelah win 
10. martinggale win 
11. maksimal martinggale win 
untuk lebih jelas dapat di nonton tutorial dan penjelasan cara mengunakanya.
 https://youtu.be/fCHRaUSOQg8  

info saidforex26@gmail.com 
facebook https://www.facebook.com/lasaid.haril...

**Comment by rickycalame** [comment]
kak @delano mau dong kak tes botnya ini email saya ya kak info.rickycalame@gmail.com
thx u kak

**Comment by kkptra** [comment]
Hi, i have some new MM tricks to test... Can anyone share the bot at kkptra@gmail.com

Thanks

**Comment by freedonone** [comment]
Gan Boleh coba bot nya Send pocation001@gmail.com please gan

**Comment by XCelestialX** [comment]
Can you share the bot? crist.w99@gmail.com

**Comment by DaveCortes21** [comment]
Can you share it, please? smarcky@gmail.com

**Comment by Craziestar36** [comment]
Kindly send it to craziestar36@gmail.com.
I will try help you create your bot.

**Comment by Rifqi0999** [comment]
Q bantuin. Kirim bot nya gan..  rifqi

**Comment by DaveCortes21** [comment]
smarcky@gmail.com

**Comment by dozietheg** [comment]
pls send to me too 
dozietheg@gmail.com

**Comment by claudiocesar10** [comment]
Send to me, please - claudiocesar10@gmail.com

**Comment by argatafx** [comment]
@delano24 halo om,,bisa dikirim sample botnya om ,,mau edit mm nya
kirim argatafx@gmail.com

**Comment by aliherman** [comment]
plase send to 4lylub1s@gmail.com

**Comment by alexssadro** [comment]
alansilva433@gmail.com

**Comment by isave** [comment]
lease send me the bot, if i win awill donate to you
kingtrader196@gmail.com

**Comment by GreenB0t** [comment]
@delano24 I'll take a look & work on the MM and send you back
abdessalamlh111@gmail.com

**Comment by InsanPangarep** [comment]
Please send to bot.binary.me@gmail.com

**Comment by tymu** [comment]
https://www.youtube.com/watch?v=NqLGLvv1Ceo
